Jint/Native/JsSet.cs

```diff
@@ -47,17 +47,17 @@ protected override bool TryGetProperty(JsValue property, [NotNullWhen(true)] out
         return base.TryGetProperty(property, out descriptor);
     }
 
-    internal void Add(JsValue value) => _set.Add(value);
+    public void Add(JsValue value) => _set.Add(value);
 
-    internal void Remove(JsValue value) => _set.Remove(value);
+    public void Remove(JsValue value) => _set.Remove(value);
 
-    internal void Clear() => _set.Clear();
+    public void Clear() => _set.Clear();
 
-    internal bool Has(JsValue key) => _set.Contains(key);
+    public bool Has(JsValue key) => _set.Contains(key);
 
-    internal bool SetDelete(JsValue key) => _set.Remove(key);
+    public bool SetDelete(JsValue key) => _set.Remove(key);
```

Comment:
`Remove` (which should be `Delete`) already handles this so can be removed altogether.

Reply:
I kept this name for now given that "Delete" is inherited. Alternatively I could use `new` to hide the inherited version although I think that's generally discouraged.